### ISSUE TITLE
Sending Tokens to Coinpayements

### ISSUE THREAD

**Issue by johnp1954** [syscoin/blockmarket-desktop-public#380]
i cannot send tokens to CP, after clicking confirm the dialog closes but thats it nothing happens
I had this issue the other day and i closed and opened it and it was then fine, that doesnt work today
The wallet is Unlocked

**Comment by btcgeek526** [comment]
Had same issue here too. Assets/tokens donot show correct balance on bm3 and this occurs when you try to send cps more than actually you have. Maybe check cps token balance on qt wallet and send that amount (minus some cps) from bm3.

**Comment by johnp1954** [comment]
Yes this was the problem, balance shown is not always correct so i was trying to send more than i had. If you check first in QT and send that amount it works.

**Comment by sidhujag** [comment]
The syscoin core should throw an error if you try to send more than you have before it lets you send the transaction to the network. Maybe bm3 not seeing that error ?

**Comment by Crcry** [comment]
Exactly, BM3 doesnt show any error messages for unsual token/asset transfering and doesnt show correct token balance even when it is fully synced. These are the main issues for now.
